### DISCUSSION TITLE
Backup Micro SD Card

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#119] [Tips]
**Backup Micro SD Card to Image File**

**Linux** 
- Move micro SD card to Linux.
- Create compressed image file:
```sh
sudo bash <( wget -qO - https://github.com/rern/rOS/raw/main/imagecreate.sh )
```
<br>
  
**Windows** 
- Move micro SD card to PC.
- [**Win32DiskImager**](https://sourceforge.net/projects/win32diskimager/)
    - Select location and name image file.
    - `Read`
- To save space, compress image file.

**Comment by BV-Labs** [comment]
/dev/sde is not rAudio.

**Reply by rern** [reply]
Should be OK now.

**Comment by BV-Labs** [comment]
Flash..!!!.... and a 32 byte image file is created.

**Comment by firstchanceongit** [comment]
I can highly recommend

https://github.com/Drewsif/PiShrink

to shrink the image (prior to compressing it with a zipper). It's especially useful if you have to burn the image later to a SD card that is (even only a tiny bit) smaller than the original SD card from which the image was made, i.e., if you used a 16GB SD card for the original installation you can later also use a 4GB or 8GB (or a 16GB card from another manufacturer which may provide slightly less space). The shrunk image will later automatically expand again to the size of the used SD card.

Unfortunately, shrinking can only be done on Linux systems. I'm not aware of any similar tool that allows to do the shrink process on a Windows PC.

**Reply by BV-Labs** [reply]
Thank you sooooo much! A very useful tool.
